test/system/journals_test.rb

```diff
@@ -28,22 +28,20 @@ def setup
   test 'can create journal entry by link button in show and then delete it' do
     login_as @user
     visit client_path(@client)
-    click_button 'Journal'
-    click_link 'Write a journal entry'
+    first('a', text: 'Journal').click
```

Comment:
Does `click_link 'Journal'` not work here? (also two times further down)

Reply:
It doesn't because there is two of these links now. (Menu on top and bottom of page)

Reply:
Ok makes sense